cellprofiler/modules/loadimages.py

```diff
@@ -79,7 +79,6 @@
 import urlparse
 
 import cellprofiler.measurement
-import numpy as np
```

Comment:
Why are we loosing the numpy import?

Reply:
they do both `import numpy` and `import numpy as np`. I can leave it or fix np versus numpy references throughout.

Reply:
I kept both for now.

Reply:
Definitely leave it and keep the changes to a minimum.